From 90ba57ad76693f845d26fc86df496ee13dc70b4d Mon Sep 17 00:00:00 2001
From: Jie Zhang <quic_jiezh@quicinc.com>
Date: Sat, 15 Feb 2025 16:13:25 +0530
Subject: [PATCH 5/5] FROMLIST: arm64: dts: qcom: qcs8300-ride: Enable Adreno
 623 GPU

Enable GPU for qcs8300-ride platform and provide path for zap
shader.

Signed-off-by: Jie Zhang <quic_jiezh@quicinc.com>
Signed-off-by: Akhil P Oommen <quic_akhilpo@quicinc.com>
Patchwork: https://patchwork.freedesktop.org/patch/637047/
Link: https://lore.kernel.org/lkml/20250213-a623-gpu-support-v1-5-993c65c39fd2@quicinc.com/
Upstream-Status: Submitted [https://lore.kernel.org/lkml/20250213-a623-gpu-support-v1-5-993c65c39fd2@quicinc.com/]
---
 arch/arm64/boot/dts/qcom/qcs8300-ride.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcs8300-ride.dts b/arch/arm64/boot/dts/qcom/qcs8300-ride.dts
index cf2fe52d02d5..dbcb2cdb4d8e 100644
--- a/arch/arm64/boot/dts/qcom/qcs8300-ride.dts
+++ b/arch/arm64/boot/dts/qcom/qcs8300-ride.dts
@@ -362,6 +362,14 @@ queue3 {
 	};
 };
 
+&gpu {
+	status = "okay";
+};
+
+&gpu_zap_shader {
+	firmware-name = "qcom/qcs8300/a623_zap.mbn";
+};
+
 &qupv3_id_0 {
 	status = "okay";
 };
-- 
2.34.1

